lib/progressive/project_decorator.rb

```diff
@@ -37,9 +35,9 @@ def issues_progress(open)
 
   # Cloned from Version#completed_percent
   def issues_completed_percent
-    if issues.count == 0
+    if issues.count.zero?
```

Comment:
Please, keep this code consistent with ‘Version#completed_percent’ method.

Reply:
done